src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<faces-config xmlns="http://xmlns.jcp.org/xml/ns/javaee"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.3"
-  xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-facesconfig_2_3.xsd">
-  <application>
-    <action-listener>org.primefaces.application.DialogActionListener</action-listener>
-    <navigation-handler>org.primefaces.application.DialogNavigationHandler</navigation-handler>
-    <view-handler>org.primefaces.application.DialogViewHandler</view-handler>
-  </application>
+<?xml version="1.0" encoding="UTF-8"?>
+<faces-config xmlns="https://jakarta.ee/xml/ns/jakartaee"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="4.0"
+  xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-facesconfig_4_0.xsd">
+  <application>
```

Comment:
Only Dark green highlighted part in namespace and header files has been changed.
Rest are same.